app/interface.py

```diff
@@ -287,6 +287,13 @@ def get_harvest_record_by_source(self, source_id):
                 HarvesterDBInterface._to_dict(rcd) for rcd in harvest_records]
             return harvest_records_data
 
+    def get_source_by_jobid(self, jobid):
+        harvest_job = self.db.query(HarvestJob).filter_by(id=jobid).first()
+        if harvest_job is None:
+            return None
+        else:
+            return HarvesterDBInterface._to_dict(harvest_job.source)
```

Comment:
is this equivalent to the line above where it's calling `self._to_dict(x)`?
ex. https://github.com/GSA/datagov-harvesting-logic/pull/57/files#diff-b42bfc923c5151b9f34311dee333cb4f4095bb81de60964cd1fd23fa28a97b4cR70

Reply:
resolved on a call to clean up inconsistencies in near future